doc/vlan/SONiC_vlan_translation_qnq.md

```diff
@@ -787,7 +787,7 @@ sonic(conf-if-Ethernet4)# switchport vlan-mapping remove 50 dot1q-tunnel 100
 
 **Syntax**
 ```
-[no] switchport vlan-mapping {<svlan-id> | dot1q-tunnel <svlan-id>} [priority [<priority-bits>]]
+[no] switchport vlan-mapping <svlan-id> [priority [<priority-bits>]]
```

Comment:
No priority bit in "no" command

Reply:
Updated the command
@smahara pls have a look